test/suite/test_timestamp03.py

```diff
@@ -215,27 +270,51 @@ def test_timestamp03(self):
             count += 1
         # print "Updated " + str(count) + " keys to value2"
 
+        # Scenario : 4
+        # Now check that we don't see the updated data of timestamp tables
+        # with old read_timestamp
+        self.check(self.session, 'read_timestamp=' + timestamp_str(100),
```

Comment:
Again reorder so that timestamp and non-timestamp tables are together and add a comment for each:
`# Tables using timestamps should only see the original value.`
...
`# Tables not using timestamps should only see value2.`